src/js/model/primitives/Primitive.js

```diff
@@ -4,18 +4,24 @@ var dl = require('datalib'),
     lookup = model.primitive,
     id = 0;
 
+/**
+ * @classdesc A Lyra Primitive.
+ *
+ * @description Lyra primitives are wrappers around corresponding Vega
+ * definitions. Lyra-specific properties are denoted with an underscore prefix
+ * and are removed during clean exports. Each Lyra primitive is given a unique
+ * identifier, and these identifiers are used to store references to other
+ * primitives.
+ *
+ * @constructor
+ */
 function Primitive() {
   model.primitive(this._id = ++id, this);
   return this;
 }
 
-var prototype = Primitive.prototype;
-
-// Primitive classes are wrappers around the corresponding Vega
-// specification. Clean them up to remove Lyra-specific things.
-// This also converts signalRefs -> values unless resolve === false.
-function clean(spec, resolve) {
-  var k, p, c, res = resolve !== false;
+function _clean(spec, clean) {
+  var k, p, c, cln = clean !== false;
```

Comment:
Good call. I thought I had incorporated this comment in the class descriptor + `export` method but looking at it now, that's not entirely true. Must've spaced last night. Will fix presently.